stable/jenkins/values.yaml

```diff
@@ -220,6 +220,10 @@ master:
 
   # Enable to always override the installed plugins with the values of 'master.installPlugins' on upgrade or redeployment.
   # overwritePlugins: true
+
+  # Enable to always override the installed plugins with the values of 'master.installPlugins' on upgrade or redeployment.
```

Comment:
The comment is misleading. The property says `keep` and the comment states `override`.
Should we rename the property to `keepPluginsFromImage`?